client/src/components/aggregations/utils.js

```diff
@@ -0,0 +1,154 @@
+import _clone from "lodash/clone"
+import _isEmpty from "lodash/isEmpty"
+import moment from "moment"
+import Settings from "settings"
+
+export const countPerDateAggregation = (fieldName, fieldConfig, data) => {
+  const values = data.reduce((counter, entity) => {
+    const dateFieldValue = Object.get(entity, fieldName)
+    const value = dateFieldValue
+      ? moment(dateFieldValue).format("YYYY-MM-DD")
+      : null
+    counter[value] = ++counter[value] || 1
+    return counter
+  }, {})
+  return { values: values }
+}
+
+// TODO: maybe use a library for a list of colors
+const CHART_COLORS = [
+  "#3366CC",
+  "#DC3912",
+  "#FF9900",
+  "#109618",
+  "#990099",
+  "#3B3EAC",
+  "#0099C6",
+  "#DD4477",
+  "#66AA00",
+  "#B82E2E",
+  "#316395",
+  "#994499",
+  "#22AA99",
+  "#AAAA11",
+  "#6633CC",
+  "#E67300",
+  "#8B0707",
+  "#329262",
+  "#5574A6",
+  "#3B3EAC"
+]
```

Comment:
Where do these colours come from?

Reply:
Was coming from here: https://gist.github.com/there4/2579834#file-chart_colors-txt
But now I've changed it with this one https://gist.github.com/mikebmou/1323655 as it is more recent.